frame/society/src/lib.rs

```diff
@@ -413,12 +430,12 @@ decl_storage! {
 		pub Rules get(fn rules): Option<T::Hash>;
 
 		/// The current set of candidates; bidders that are attempting to become members.
-		pub Candidates get(fn candidates): Vec<Bid<T::AccountId, BalanceOf<T, I>>>;
+		pub Candidates get(fn candidates): Vec<Bid<T::AccountId, BalanceOf<T, I>, OpaqueCall>>;
```

Comment:
```suggestion
		pub Candidates get(fn candidates): Vec<Bid<T::AccountId, BalanceOf<T, I>, <T as frame_system::Config>::Call>>;
```

Why not storing the call itself? it is encodable.

Reply:
Honestly, I was having some obscure and hard to resolve errors with from/into when actually dispatching the Call - this was just my way to get the pallet to build and I haven't been able to get this working without the OpaqueCall type. - added this change but I'm reverting it now

Reply:
maybe I can also give it a go, if you have specific compilation issues?

Reply:
would be welcome if you have the time - my philosophy here is that it works and isn't difficult to understand so I'm not going to mess with it too much 😅